docker-rtail/README.md

```diff
@@ -35,8 +35,8 @@ WATCH_IREGEX='.*catalina\\.out\\\|.*\\.log'
 # Optionally specify -e WATCH_IREGEX to set log pattern for watching for changes in the log directory. Defaults to any path ending in .log
 #
 $docker run -dit --name=rtail --link rtail-server -v <path to log directory>:/logs maluuba/rtail
-# For remote connectivity, use
-$docker run -dit --name=rtail ---add-host=rtail-server:${RTAIL_SERVER_IP} -v <path to log directory>:/logs maluuba/rtail
+# For remote connectivity, use the following:
+$docker run -dit --name=rtail -e "RTAIL_PREFIX=server_name/" --add-host=rtail-server:${RTAIL_SERVER_IP} -v <path to log directory>:/logs maluuba/rtail
```

Comment:
RTAIL_PREFIX is different than ID_PREFIX in the script.
Also in the readme maybe have a separate line explaining what the flag does, like 'Optionally specify -e ... to add a prefix to the rtail channel ID'